utils/Time.h

```diff
@@ -66,6 +66,26 @@ constexpr bool diffGE(const uint64_t a, const uint64_t b, const uint64_t value)
     return static_cast<int64_t>(b - a) >= static_cast<int64_t>(value);
 }
 
+constexpr bool lt(const uint64_t a, const uint64_t b)
```

Comment:
Add methods to compare two timestamps

Reply:
Not that it was correct before, but all of these could and should be based on the diff method at line 44.
including diffLT, diffLE.

Reply:
I changed the `diff...` to use `diff`. This ones uses the `diff...`